src/alitra/align_frames.py

```diff
@@ -57,18 +57,15 @@ def align_frames(
             from_=p_1.frame,
             to_=p_2.frame,
         )  # type: ignore
-
-        euler: Euler = Euler.from_array(
-            rotation_object.as_euler("zyx"),
+        transform = Transform(
             from_=p_1.frame,
             to_=p_2.frame,
+            translation=translations,
+            rotation_object=rotation_object,
         )
-        euler = AlignFrames._set_non_rotating_angles_to_zero(euler, rot_axes)
```

Comment:
Is it correct to say that the function used here previously meant that it was only possible to rotate along one axis?

And now we may specify rotations on any axis?

Reply:
Due to estimation errors, the rotation of an axis that should not be rotated of might be non-zero. This function was imposed to enforce that these should be zero. For example, if there is a rotation [0,0,1] about the z-axis, the procedure might find [1e-3,1e-4,1], which was modified to [0,0,1] with this function. 

If this is a good solution is questionable. For the previous example,  the function should not be needed if the two input frames has the same z-coordinates for each point.  In this case I think it would be better to enforce it by checking that the z-coordinate does not change when rotating about the z-axis. Might add this in a new PR.